frame/frame.go

```diff
@@ -624,6 +624,14 @@ func (frame *Frame) correlate() {
 	}
 }
 
+// SampleNumber returns the first sample number embedded in a frame.
```

Comment:
Just a suggestion for phrasing, they are equivalent.

```suggestion
// SampleNumber returns the first sample number contained within the frame.
```